src/ripple/json/impl/json_reader.cpp

```diff
@@ -141,7 +143,7 @@ Reader::readValue(unsigned depth)
 {
     Token token;
     skipCommentTokens ( token );
-    if (++depth >= 1000)
+    if (++depth >= nest_limit)
```

Comment:
Should this be `if (depth > nest_limit)` ?
`depth` is already incremented by the `readObject` and `readArray` functions before `readValue` is called. Incrementing here seems to incorrectly double how many nests we think there are. With following json ```{"obj":{"obj":{"obj":{"obj":{"obj":{"obj":{"obj":{"obj":{"obj":{"obj":{}}}}}}}}}}}```  depth reaches a value of 20 instead of 10.

Reply:
<slaps forehead> Thanks!  Fixing...